editor/code_editor.cpp

```diff
@@ -2086,6 +2086,7 @@ CodeTextEditor::CodeTextEditor() {
 	text_editor = memnew(CodeEdit);
 	add_child(text_editor);
 	text_editor->set_v_size_flags(SIZE_EXPAND_FILL);
+	text_editor->set_structured_text_bidi_override(TextServer::STRUCTURED_TEXT_GDSCRIPT);
```

Comment:
Shouldn't this be set it `ScriptTextEditor` if it's `GDScript`, otherwise this will affect all scripts and the shader editor?